app/views/admin/responses/_form.html.erb

```diff
@@ -0,0 +1,47 @@
+<%= form_for [:admin, consultation, consultation_response], url: [:admin, consultation, consultation_response.singular_routing_symbol] do |form| %>
+  <%= render "govuk_publishing_components/components/fieldset", {
+    legend_text: "Published on",
+    heading_size: "s"
+  } do %>
+    <%= render "components/datetime_fields", {
+      prefix: "consultation_#{consultation_response.singular_routing_symbol}",
+      field_name: "published_on",
+      # date_heading: "Published on",
+      date_only: true,
+      year: {
+        value: consultation_response.published_on.year,
+        start_year: 1997,
+        end_year: Date.today.year + 5
+      },
+      month: {
+        value: consultation_response.published_on.month
+      },
+      day: {
+        value: consultation_response.published_on.day
```

Comment:
if we assign this the id of `"consultation_#{consultation_response.singular_routing_symbol}_published_on"` the errors will link and focus on the day input

Reply:
👍